File: playwright-tests/javascript/playwright_test.js
const { chromium } = require('playwright');

(async () => {
    const browser = await chromium.launch({ headless: false });
    const page = await browser.newPage();

    await page.goto('https://www.wikipedia.org/');
    await page.click('text=English');
    await page.waitForURL('https://en.wikipedia.org/wiki/Main_Page');

    const title = await page.title();
    console.log(`Página actual: ${title}`);

    await browser.close();
})();

File: playwright-tests/python/playwright_test.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch(headless=False)
    page = browser.new_page()
    
    page.goto("https://www.bing.com")
    page.fill("input[name='q']", "Playwright automation")
    page.press("input[name='q']", "Enter")
    
    page.wait_for_selector("h2 a")
    results = page.locator("h2 a").all_text_contents()
    print("Resultados:", results[:5])

    browser.close()
